### PR TITLE
handle additional cgroup file locations

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[testonlyextended][core(cgroup)]

**Comment by bparees** [comment]
@csrwng ptal

File: pkg/build/builder/util.go [hunk 1]
```diff
@@ -74,26 +74,40 @@ func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
...
+	// to /sys/fs/cgroup/cpu,cpuacct, so technically the next check
+	// would be sufficient, but it seems better to rely on the real directory
+	// rather than a symlink.
 	if _, err := os.Stat("/sys/fs/cgroup/cpu,cpuacct"); err == nil {
-		cgroupDir = "/sys/fs/cgroup/cpu,cpuacct"
+		cpuDir = "/sys/fs/cgroup/cpu,cpuacct"
 	}
 
-	cpuQuota, err := readInt64(filepath.Join(cgroupDir, "cpu.cfs_quota_us"))
+	// seen on docker1.10 fedora systems
```
Comment (csrwng) [inline pkg/build/builder/util.go]: so I installed 1.10 on fedora and it does have cpu,cpuacct. So actually this is seen in debian-like distros like TinyCore linux in boot2docker.

**Comment by csrwng** [comment]
Just one comment, other than that LGTM

**Comment by bparees** [comment]
@csrwng updated the comment, thanks.
[merge]

**Comment by bparees** [comment]
extended test failures are due to: https://github.com/openshift/origin/pull/7200

**Comment by smarterclayton** [comment]
Travis thinks you broke go 1.4.  Prove it wrong!

**Comment by bparees** [comment]
continuous-integration/travis-ci/pr The Travis CI build passed
?

**Comment by smarterclayton** [comment]
https://travis-ci.org/openshift/origin/builds/108430935

**Comment by bparees** [comment]
```
pkg/serviceaccounts/client.go:40: declaration of err shadows declaration at pkg/serviceaccounts/client.go:32: 

pkg/serviceaccounts/controllers/create_dockercfg_secrets.go:247: declaration of err shadows declaration at pkg/serviceaccounts/controllers/create_dockercfg_secrets.go:239: 

pkg/serviceaccounts/controllers/docker_registry_service.go:185: declaration of err shadows declaration at pkg/serviceaccounts/controllers/docker_registry_service.go:155: 

pkg/serviceaccounts/controllers/docker_registry_service.go:192: declaration of err shadows declaration at pkg/serviceaccounts/controllers/docker_registry_service.go:185: 
```

if this PR did that, i have magical powers...  but i'll clean it up in another PR.

**Pull request merged**